src/main/java/dk/cs/aau/huppaal/presentations/JorkPresentation.java

```diff
@@ -27,7 +27,7 @@ public class JorkPresentation extends Group implements SelectHelper.Selectable {
     private final JorkController controller;
 
     public JorkPresentation(@NamedArg("type") final String type) {
-        this(new Jork(type.equals("JOIN") ? Jork.Type.JOIN : Jork.Type.FORK), null);
+        this(new Jork(type.equals("JOIN") ? Jork.Type.JOIN : Jork.Type.FORK, null), null);
     }
```

Comment:
Is this for the help dialog?

Reply:
I think so? 🤔 My IntelliJ says the `JorkPresentation` constructor is unused though. Should we just remove the ctor?